### ISSUE TITLE
Frame count comparison for GIF (and other video formats)

### ISSUE THREAD

**Issue by waffle2028** [arsenetar/dupeguru#1013]
**Is your feature request related to a problem? Please describe.**
Using picture comparison mode, there are not enough properties for the comparison of GIF files. Everything else is similar but the frame count is different; which I don't what to mark as a duplicate.

**Describe the solution you'd like**
I think comparing frame counts will be useful for filtering.

**Describe alternatives you've considered**
It might be better to ignore the files with different frame counts, instead of using it for the "Filter Hardness". Adding the "Don't compare different frame counts" checkbox in the Options might be helpful.

**Additional context**
I'm using version 4.2.1

**Comment by glubsy** [comment]
Yes, adding number of frames for GIF was something I wanted to work on. CF. https://doc.qt.io/qt-5/qimagereader.html#imageCount
